matlab/src/Util.h

```diff
@@ -60,4 +60,32 @@ void* createShared(std::shared_ptr<T> p)
     return new std::shared_ptr<T>(std::move(p));
 }
 
+inline void* createProxy(Ice::ObjectPrx p)
+{
+    return new Ice::ObjectPrx(std::move(p));
+}
+
+inline void* createProxy(std::optional<Ice::ObjectPrx> p)
+{
+    return p ? createProxy(std::move(p).value()) : nullptr;
+}
+
+inline Ice::ObjectPrx restoreProxy(void* p)
```

Comment:
Maybe derefProxy / derefNullableProxy would be more consistent with the `deref<T>` function name?

Reply:
I'm fine either way.